apps/server/src/form-instances/form-instances.service.ts

```diff
@@ -62,19 +64,22 @@ export class FormInstancesService {
       );
     }
 
-    // all positions in signatures should be valid
-    const positionIds = new Set(
-      createFormInstanceDto.signatures.map(
-        (signature) => signature.signerPositionId,
-      ),
-    );
-    const positions = await this.positionService.findAllWithIds(
-      createFormInstanceDto.signatures.map(
-        (signature) => signature.signerPositionId,
-      ),
-    );
-    if (positions.length != positionIds.size) {
-      throw Error(PositionsErrorMessage.POSITION_NOT_FOUND);
+    // check that the assigned user is a valid employee and the user's position is a valid position
+    for (let i = 0; i < createFormInstanceDto.signatures.length; i++) {
+      const userId = createFormInstanceDto.signatures[i].assignedUserId;
+      const positionId = createFormInstanceDto.signatures[i].signerPositionId;
+
+      const position = await this.positionService.findOne(positionId);
```

Comment:
Did you test the case where we are unable to find the position and it's throwing the error properly? Code wise looks good just want to make sure.
Also, really picky, but can we remove the empty lines on 73 and 79.